### DISCUSSION TITLE
Units in generating stl files for xcompact3D toolbox

### DISCUSSION THREAD

**Discussion by Sushmitharao46** [xcompact3d/Incompact3d#190] [Q&A] [answered]
I'm trying to get experienced with the sandbox configuration for which I've started with reproducing the cylinder case. what are the dimensions that are to be considered while generating the **stl** files are they in **mm or meters** ? 
Thank You.

**Comment by nbeb** [comment, marked as answer]
Hi, 
By default, the stl file is expected to have dimensions that are consistent with what you define in Incompact3d's input file (e.g., domain size, consideration (or-not) of non-dimensional units etc). However, you can use the scale option in .geo.from_stl where you can rescale everything to match your problem size (i.e. you can use the same stl to immerse objects in domains of different size).
